app/controllers/api/patients_controller.rb

```diff
@@ -1,13 +1,25 @@
 class Api::PatientsController < Api::BaseController
   before_action :authenticate_token
-  before_action :create_zip, only: [:get_results]
-
+  before_action :create_json, only: [:get_results]
+  
```

Comment:
You didn't assign the new patient to group

Reply:
I have assigned it in the create function directly.

Reply:
To be clear, the group functionality is not added to the api yet, I would add it once the feature is finished. I have already added the assignment of new patients to the non api controller is what I mean.

Reply:
If you didn't finish it, say it in your status email. Don't mislead others.